src/components/select-field/__tests__/BaseSelectField.test.js

```diff
@@ -1023,6 +1067,24 @@ describe('components/select-field/BaseSelectField', () => {
             instance.selectSingleOption(index);
         });
 
+        test('should call handleChange with the correct item when searchText is not empty string', () => {
+            const indexBeforeFilter = 1; // Index of audio option in default options array
+            const indexAfterFilter = 0; // Index of audio option after user inputs a search string of 'Audio'
+            const wrapper = shallowRenderSelectField();
+            const instance = wrapper.instance();
+
+            instance.setState({
+                searchText: 'Audio',
+            });
+
+            sandbox
```

Comment:
We're moving away from Sinon mocks in BUIE. Can we use Jest mocks for the new tests in this file?

Reply:
Addressed!